hoodie-common/src/main/java/com/uber/hoodie/common/table/timeline/HoodieActiveTimeline.java

```diff
@@ -51,14 +52,19 @@
 
   public static final SimpleDateFormat COMMIT_FORMATTER = new SimpleDateFormat("yyyyMMddHHmmss");
 
+  //TODO: Ask @nishith what does it take to treat rollback extensions. Is it enough if we start archiving Rollback
+  // timeline
+  public static final Set<String> VALID_ROLLBACK_EXTENSIONS = new HashSet<>(Arrays.asList(INFLIGHT_ROLLBACK_EXTENSION,
```

Comment:
You want to present them in the timeline only for archival ? If yes, this works fine, if you want it to surface in all timelines, we need to test it thoroughly since this tampers with the timeline result